src/globals.js

```diff
@@ -275,23 +275,27 @@ for (const key in geoKeyNames) {
 }
 
 export let parseXml; // eslint-disable-line
+export let UNORDERED_NODE_SNAPSHOT_TYPE; // eslint-disable-line
```

Comment:
I think it is unnecessary to make a big deal out of the retrieval of this constant. As far as I can see this constant is derived from the XPath standard (https://www.w3.org/TR/DOM-Level-3-XPath/xpath.html#XPathResult-UNORDERED-NODE-SNAPSHOT-TYPE), so I think it is enough to say `export const UNORDERED_NODE_SNAPSHOT_TYPE  = 6;`

Reply:
Couldn't agree more! :-)